app/api/reducers/dataReducer.js

```diff
@@ -55,6 +55,11 @@ function dataReducer(state = initialState, action) {
       return handleData(state, { resources });
     }
 
+    case actionTypes.RESERVATION_DELETE_SUCCESS: {
+      const reservation = Object.assign({}, action.payload, { state: 'cancelled' });
```

Comment:
- [ ] Doesn't the API return cancelled state? Can we receive later on cancelled reservations from the backend?